lm_eval/models/openai_completions.py

```diff
@@ -470,39 +505,32 @@ def sameuntil_chunks(xs, size):
                 contexts, all_gen_kwargs = zip(*chunk)
                 inps = [{"role": "user", "content": context} for context in contexts]
 
-                gen_kwargs = all_gen_kwargs[0]
+                context_gen_kwargs = all_gen_kwargs[0]
                 until = None
-                if isinstance(gen_kwargs, dict):
-                    kwargs = copy.deepcopy(gen_kwargs)  # edge case for repeats > 1
+                if isinstance(context_gen_kwargs, dict):
+                    kwargs = copy.deepcopy(
+                        context_gen_kwargs
+                    )  # edge case for repeats > 1
                     if "until" in kwargs.keys():
                         until = kwargs.pop("until")
                         if isinstance(until, str):
                             until = [kwargs]
                         elif not isinstance(until, list):
                             raise ValueError(
-                                f"Expected `kwargs['until']` to be of type Union[str,list] but got {until}"
+                                f"Expected repr(kwargs['until']) to be of type Union[str, list] but got {until}"
                             )
                 else:
                     raise ValueError(
-                        f"Expected `kwargs` to be of type `dict` but got {kwargs}"
+                        f"Expected repr(kwargs) to be of type repr(dict) but got {kwargs}"
                     )
 
-                if "max_gen_toks" in kwargs.keys():
-                    max_gen_toks = kwargs.pop("max_gen_toks")
```

Comment:
These ended up not being called directly anymore and flake8 was complaining

Reply:
yup, makes sense, this should be removed now since it gets piped directly in!